MediaBrowser.MediaEncoding/Probing/ProbeResultNormalizer.cs

```diff
@@ -1027,27 +1027,32 @@ private bool IsClose(double d1, double d2, double variance = .005)
         /// </summary>
         /// <param name="value">The value.</param>
         /// <returns>System.Nullable{System.Single}.</returns>
-        private float? GetFrameRate(string value)
+        internal static float? GetFrameRate(ReadOnlySpan<char> value)
         {
-            if (string.IsNullOrEmpty(value))
+            if (value.IsEmpty)
             {
                 return null;
             }
 
-            var parts = value.Split('/');
-
-            float result;
-
-            if (parts.Length == 2)
+            int index = value.IndexOf('/');
+            if (index == -1)
             {
-                result = float.Parse(parts[0], CultureInfo.InvariantCulture) / float.Parse(parts[1], CultureInfo.InvariantCulture);
+                // REVIEW: is this brach actually required? (i.e. does ffprobe ever output something other then a fraction?)
```

Comment:
Does it matter? Better safe than sorry.
```suggestion
```

Reply:
It's an extra branch to cover in our tests :p

Reply:
Unless you can prove that it's unnecessary I say leave it in

Reply:
That's what the comment is for...
The code stays until I'm sure it isn't needed

Reply:
Then change the comment to reflect that ;) And fix the typos and spelling errors